### PR TITLE
Print exception that may have occurred during test execution

### PR DESCRIPTION
**Pull request by ioana-blue**
Added missing catch to not lose info on potential exceptions that happened during test and are not logged elsewhere.

### CODE REVIEW

**Comment by ioana-blue** [comment]
Due to the nature of the change, I request a PG-exemption. But let me know if you think a PG is needed.

**Comment by markusthoemmes** [comment]
What are we gaining with this? Shouldn't the exception go uncaught anyway and thus be logged by gradle/JUnit whomever?

**Comment by rabbah** [comment]
Yes except there could be two exceptions and the last masks the first. Here was the thought behind this maybe it leads to something better. Suppose a create failed within the asset helper - this would throw an exception which is caught by the try. But then the cleaner throws an exception also because the delete will fail as well. So you don't see the exception from the create.

File: tests/src/common/WskTestHelpers.scala [hunk 1]
```diff
@@ -80,6 +80,11 @@ trait WskTestHelpers extends Matchers {
 
         try {
             test(wskprops, new AssetCleaner(assetsToDeleteAfterTest, wskprops))
+        } catch {
+            case t: Throwable =>
+                // log the exception that occurred in the test
+                println("Exception occurred during test execution:")
+                println(t)
```
Comment (markusthoemmes) [inline tests/src/common/WskTestHelpers.scala]: Stacktrace not needed/helpful?

**Comment by rabbah** [comment]
Merged as https://github.com/openwhisk/openwhisk/commit/4f24aad8985c3f8370c4aa1ece8920c810500c1a.

**Pull request closed without merging**